capstone-frontend/src/components/ClubPage.jsx

```diff
@@ -1,23 +1,72 @@
 import React, { Component } from 'react';
 
+import BookSearchTile from './BookSearchTile';
+import AssignmentCard from './AssignmentCard';
+ 
+import '../styles/Groups.css';
+
 
 class ClubPage extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      club: {}
+      club: {},
+      book: {},
+      assignments: [],
+      owner: {}, 
+      members: [], 
+      bookLists: []
     }
   }
 
-  fetchClubData = () => {
-    fetch(`/api/clubs?id=${this.props.match.params.id}`)
-        .then(response => response.json()).then(res => this.setState({club: res[0]}))
+  fetchData = async () => {
+    await fetch(`/api/clubs?id=${this.props.id}`)
```

Comment:
Nice job integrating calls to several different servlets in a consistent manner to retrieve data for the ClubPage!

Reply:
Thanks!